banzai/stages.py

```diff
@@ -211,16 +211,19 @@ def apply_master_calibration(self, images, master_calibration_image, logging_tag
             qc_results = {"master_comparison.fraction": bad_pixel_fraction,
                           "master_comparison.snr_threshold": self.SIGNAL_TO_NOISE_THRESHOLD,
                           "master_comparison.pixel_threshold": self.ACCEPTABLE_PIXEL_FRACTION,
-                          "master_comparison.failed": frame_is_bad,
                           "master_comparison.comparison_master_filename": master_calibration_image.filename}
 
             for qc_check, qc_result in qc_results.items():
                 logs.add_tag(logging_tags, qc_check, qc_result)
             logs.add_tag(logging_tags, 'filename', image.filename)
             logs.add_tag(logging_tags, 'master_comparison_filename', master_calibration_image.filename)
+
             self.logger.info("Performing comparison to last good master {caltype} frame".format(
                 caltype=self.calibration_type), extra=logging_tags)
```

Comment:
While you are here, this is weird spacing. It would be better to define a message variable and then call format in a separate line.